cranelift/codegen/src/isa/aarch64/inst.isle

```diff
@@ -935,7 +935,13 @@
 
        ;; A dummy use, useful to keep a value alive.
        (DummyUse
-        (reg Reg))))
+        (reg Reg))
+
+       ;; Emits a inline stack probe loop.
```

Comment:
s/a/an/

Also can we note here that it's careful not to use spilltmp or tmp2, so these registers are allowed to be inputs?